css/main.css

```diff
@@ -1,3 +1,673 @@
+    justify-content: space-between;
+    align-content: center;
+    font-size: var(--fm-1rem);
+}
+
+.Column {
+    display:flex;
+    flex-direction: column;
+}
+
+.ALink {
+    color: var(--white);
+    padding-bottom: var(--fm-1em);
+}
+
+ .ALink a:link {
+    text-decoration: none;
+}
+  
+.ALink a:visited {
+    text-decoration: none;
+}
+  
+.ALink a:hover {
+    text-decoration: none;
+}
+  
+.ALink a:active {
+    text-decoration: none;
+}
+
+.email-control {
+    width:40%;
+    padding-bottom: var(--fm-1em);
+    margin-right: var(--fm-1em);
+    background-color: var(--dark-green);
+    border-bottom: var(--fs-2px) solid var(--white);
+}
+
+.form-container {
+    grid-area:form;    
+    margin-bottom: var(--fm-4em);
+}
+
+.form-container p  {
+    font-size: var(--fm-1p8rem);
+    margin-bottom: var(--fm-1em);
+}
+
+.form-container input[type=submit] {
+        background-color: var(--dark-green);
+        color: var(--white);
+        padding: var(--fs-12px) var(--fs-20px);
+        border: var(--fs-2px) solid var(--white); 
+        margin-bottom: var(--fm-2em);
+        cursor: pointer;
+}
+
+.serving-container { 
+    grid-area:serving;
+    font-size: var(--fm-1p8rem);
+    color: var(--white);
+    margin-bottom: var(--fm-2em);
+}
+
+.serving-container a {
+    color: var(--white);
+}
+
+/* I adopted the answer from https://stackoverflow.com/questions/4861224/how-to-use-css-to-surround-a-number-with-a-circle */
+
+.JBLogo {
+    grid-area: logo;
+    border-radius: 50%;
+    width: var(--fm-5rem);
+    height: var(--fm-5rem);
+    padding-top:0.5em;
+    border: var(--fs-2px) solid var(--white);
+    color: var(--white);
+    text-align: center;
+    font-size: var(--fm-1p3rem);
+    font-weight:bold;
+    margin-bottom: var(--fm-1em);
+}
+
+.copyright {
+    grid-area:copyright;
+    margin-top: var(--fm-2em);
+    font-size: var(--fm-1p2rem);
+    color: var(--white);
+    margin-bottom: var(--fm-2em);
+}
+
+
+  /* MEDIA QUERY FOR LARGER DISPLAY: */
+  @media screen and (min-width: 769px) {
+
+
+    .OrangePict {
+        display:none;
+    }
+
+    .Desktop-OrangePict {
+        grid-area: orange;
+        width: 100%;
+        display:block;
+    }
+
+    .tenpercent-para {
+        margin: 0 auto;
+        display:flex;
+        justify-content: center;
+    }
+
+
+    .display-3icons {
+        flex-direction: row;
+        align-items: center;
+        gap: var(--fm-1em);
+    }
+
+    .flex-3icons {
+        display: block;
+        margin-left:8rem;
+    }
+
+
+    .GreenArea {
+        grid-area:green;
+        display: grid;
+        grid-template-columns: 1fr 1fr;
+        grid-template-rows: 1fr 1fr 1fr;
+        grid-template-areas: 
+        "greenpict greenhr"
+        "greenpict greenhr"
+        "greenpict greenbutton";
+        background-color: var(--light-green);
+    }
+
+
+    .row-flexdisplay {
+        display:flex;
+        flex-direction:row;
+        justify-content: space-between;
```

Comment:
space-around is actually better here, so that we don't have so much space in between